src/BuiltInTools/BrowserRefresh/HostingStartup.cs

```diff
@@ -23,14 +23,22 @@ public Action<IApplicationBuilder> Configure(Action<IApplicationBuilder> next)
         {
             return app =>
             {
-                app.Map("/_framework/clear-browser-cache", app1 => app1.Run(context =>
+                app.Map(Urls.ClearSiteData, app1 => app1.Run(context =>
                 {
                     // Scoped css files can contain links to other css files. We'll try clearing out the http caches to force the browser to re-download.
                     // See https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Clear-Site-Data#directives
-                    context.Response.Headers["Clear-site-data"] = "\"cache\"";
+                    context.Response.Headers["Clear-Site-Data"] = "\"cache\"";
```

Comment:
This doesn't work in Safari. See [here](https://www.caniuse.com/?search=Clear-Site-Data) maybe we could try an alternative where if we are using hot reload we plug-in on the file options and apply `Cache-control: no-cache` to all files?

Reply:
Interesting. I think we had wanted to avoid changing headers on arbitrary requests. The concern was that we might affect caching semantics in your app. We still apply a nonce to all of the css files that we re-request but not having Clear-Site-Data would break imported files.

I'll file an issue for this.